### PR TITLE
Add Create Event Feature to UG

### PR DESCRIPTION
**Pull request by jordankanghm**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
```
Comment (brein62) [inline docs/UserGuide.md:71]: explaning

File: docs/UserGuide.md [hunk 2]
```diff
@@ -151,6 +151,26 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Create an event
```
Comment (brein62) [inline docs/UserGuide.md:154]: LGTM!

**Review by brein62 — APPROVED** [review]
LGTM!

**Review by ruo-x — APPROVED** [review]
LGTMLTGMLTGLMLGTMLGTM

**Pull request merged**